conf/environments.js

```diff
@@ -102,6 +102,11 @@ module.exports = {
             ecmaVersion: 6
         }
     },
+    es7: {
```

Comment:
@ljharb who says there will be environments for those? Who says that's what they will be named?

The `es6` environment is a bit of a leftover hack due to the 20 or so individual features we used to have that people kept messing up. We kept it for back compat even when we eliminated all those flags. 

I really don't want to continue down this path, especially where we will have an undefined behavior when es6 is false and es7 is true. What's expected then?

We need a better approach than continuing to create environments, so I still think we should not have an es7 environment. If I'm wrong, we can always add it later, but we won't be able to remove it after it's released if I'm right.